pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -863,6 +863,14 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 					PropertyName: "apiUrl",
 					Required:     true,
 				},
+				{
+					Label:        "Description",
+					Description:  "A description of the incident.",
```

Comment:
I am not sure "incident" makes sense here... Probably just "message" or "description"

Reply:
Maybe mimic with what Slack uses? "The description field of the alert"?